src/sql/migrations.rs

```diff
@@ -911,6 +911,11 @@ CREATE INDEX msgs_status_updates_index2 ON msgs_status_updates (uid);
         .await?;
     }
 
+    if dbversion < 111 {
+        sql.execute_migration("DROP TABLE sending_domains", 111)
```

Comment:
You wasn't going to drop this table in order to keep backup compatibility. Or is there anything else that breaks it anyway?